file-formats/srvb/srvb.json

```diff
@@ -0,0 +1,174 @@
+            "ABAP for Key Users",
+            "ABAP Cloud Development"
+          ],
+          "enumDescriptions": [
+            "Standard",
+            "ABAP for key user extensibility",
+            "ABAP cloud development"
+          ],
+          "default": "standard"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "description",
+        "originalLanguage"
+      ]
+    },
+    "bindTypeImpl": {
+      "title": "Bind Type Impl",
+      "description": "Service Binding Implementation Name",
+      "type": "string",
+      "maxLength": 40
+    },
+    "bindType": {
+      "title": "Bind Type",
+      "description": "Service Binding Type",
+      "type": "string",
+      "maxLength": 30
+    },
+    "bindTypeCategory": {
+      "title": "Binding Category",
+      "description": "Service Binding Category",
+      "type": "string",
+      "maxLength": 1
+    },
+    "bindTypeVersion": {
+      "title": "Bind Type Version",
+      "description": "Service Binding Version",
+      "type": "string",
+      "maxLength": 30
+    },
+    "services": {
+      "title": "Services",
+      "description": "Services",
+      "type": "array",
+      "items": {
+        "title": "Service Properties",
+        "description": "Service properties",
+        "type": "object",
+        "properties": {
+          "serviceName": {
+            "title": "Service Name",
+            "description": "Service Name",
+            "type": "string",
+            "maxLength": 40
+          },
+          "serviceContent": {
+            "title": "Service Content",
+            "description": "Service Content",
+            "type": "array",
+            "items": {
+              "title": "Service Content",
+              "description": "Service content",
+              "type": "object",
+              "properties": {
+                "serviceVersion": {
+                  "title": "Service Major Version",
+                  "description": "Service Major Version",
+                  "type": "string",
+                  "maxLength": 4
+                },
+                "serviceMinorVersion": {
+                  "title": "Service Minor Version",
+                  "description": "Service Minor Version",
+                  "type": "string",
+                  "maxLength": 10,
+                  "pattern": "[0-9]*"
+                },
+                "servicePatchVersion": {
+                  "title": "Service Patch Version",
+                  "description": "Service Patch Version",
+                  "type": "string",
+                  "maxLength": 10,
+                  "pattern": "[0-9]*"
+                },
+                "serviceBuildVersion": {
+                  "title": "Service Build Version",
+                  "description": "Service Build Version",
+                  "type": "string",
+                  "maxLength": 10,
+                  "pattern": "[0-9]*"
+                },
+                "releaseState": {
```

Comment:
can this be an enum?

Reply:
No, as this is not the release state of Service Binding but for the underlying objects.